command/genesis/predeploy/params.go

```diff
@@ -75,6 +92,12 @@ func (p *predeployParams) initPredeployAddress() error {
 	return nil
 }
 
+func (p *predeployParams) initDeployerAddress() error {
+	p.deployerAddr = types.StringToAddress(p.deployerAddrRaw)
```

Comment:
Why not use the `types.IsValidAddr`?

Reply:
Because i didn't rebase my branch on develop, which contains changes for `isValidAddr`, i will do that.